packages/cli/test/dao/dao-upgrade.test.js

```diff
@@ -0,0 +1,14 @@
+import test from 'ava'
+//
+import parseCli from '../parseCli'
+
+test.serial('upgrades an app', async t => {
+  const date = new Date().getTime()
+  const id = `newdao${date}`
+
+  await parseCli(['dao', 'new', '--debug', '--aragon-id', id])
+  await parseCli(['dao', 'install', id, 'vault', '--debug'])
+  const stdout = await parseCli(['dao', 'upgrade', id, 'vault', '--debug'])
```

Comment:
Are we able to upgrade even if we are on the latest version? 🤔 Otherwise this might be a smoke test. Maybe a better way to test it e2e should be to first publish a new version and then upgrade?

Reply:
Yes it has the same behavior in the CLI as upgrading to a newer version :)